src/patterns/data/default/entity_attribute.tsv

```diff
@@ -412,7 +401,7 @@ OBA:VT0002846	aorta capacity	UBERON:0000947	aorta	PATO:0000918	volume
 OBA:VT0002847	glomerular filtration trait	GO:0003094	glomerular filtration	PATO:0001236	process quality			
 OBA:VT0002876	thyroid gland physiology trait	UBERON:0002046	thyroid gland	PATO:0001509	functionality			
 OBA:VT0002909	adrenal gland physiology trait	UBERON:0002369	adrenal gland	PATO:0001509	functionality			
-OBA:VT0002918	paired-pulse facilitation trait	CHEBI:50459	2,5-diphenylfuran	PATO:0001629	aggregated			
+OBA:VT0002918	evoked neurotransmitter secretion attribute	GO:0061670	evoked neurotransmitter secretion	PATO:0001236	process quality	paired-pulse facilitation trait		https://orcid.org/0000-0001-8314-2140
```

Comment:
Nothing this as a seemingly extreme change (I am sure you thought of this)

Reply:
The original (?) EQ looked like an error of some kind. There is no documented relationship in the scientific literature [(or in the original VT:0002918 definition)](https://www.ebi.ac.uk/ols4/ontologies/vt/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FVT_0002918) between "paired-pulse facilitation" and any kind of "biomarker" or other role for "2,5-diphenylfuran".